test/meson.build

```diff
@@ -10,7 +10,8 @@ tests = [
     'uuid',
     'template',
     'iterator',
-    'find'
+    'find',
+    'compression'
```

Comment:
I think the `compression` test needs to be conditionally built if `zstd_dep.found()`, to avoid a build failure when zstd isn't present.